api/pacman-api-compliance/src/main/java/com/tmobile/pacman/api/compliance/service/ComplianceServiceImpl.java

```diff
@@ -209,6 +199,24 @@ public Map<String, Object> getDistribution(String assetGroup, String domain) thr
         }
     }
 
+    @Override
+    public Map<String, Object> getAverageAgeDistribution(String assetGroup) throws ServiceException {
+        try {
+            Map<String, Object> distribution = new HashMap<>();
+            // get Rules mapped to targetType
+                    Map<String, Object> avgAgeDistribution = repository.getAverageAge(assetGroup);
+            logger.info("Compliance API >> Fetched avgAgeDistribution from repository: {}", avgAgeDistribution);
+
+            Map<String, Object> avgAge = repository.getAverageAge(assetGroup);
+            distribution.put("average_age_by_severity", avgAgeDistribution);

```

Comment:
Can we keep the attribute as camel case. Instead of average_age_by_severity use averageAgeBySeverity or shorten to averageAge.